test/routing/Page.test.tsx

```diff
@@ -2,24 +2,24 @@ import React from 'react';
 import { render } from '@testing-library/react';
```

Comment:
Since `Chapter` components are intended to be optional, could we add a test that ensures pages are still navigable without chapter components? Something like `test('is navigable between pages without a chapter component')`